### PR TITLE
Print fields

### PR DESCRIPTION
**Pull request by compudj**

### CODE REVIEW

File: converter/babeltrace-cfg.c [hunk 1]
```diff
@@ -1153,12 +1169,27 @@ struct bt_value *names_from_arg(const char *arg)
 		}
 	}
 
-	goto end;
+end:
+	if (found_none && found_all) {
+		printf_err("Only either \"all\" or \"none\" can be specified as argument, but not both.\n");
```
Comment (eepp) [inline converter/babeltrace-cfg.c]: How about:

    "Only either \"all\" or \"none\" can be specified in the list given to the --names option, but not both.\n"

Comment (compudj) [inline converter/babeltrace-cfg.c]: sure, do you want to provide an updated commit message too ? and then I can update the PR with those two changes.

Comment (eepp) [inline converter/babeltrace-cfg.c]: You know better than me why you made this change. Just a few words about what motivates this.

**Comment by jgalar** [comment]
Merged, thanks!

**Pull request closed without merging**